examples/webgl_materials_texture3d_volume1.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
```

Comment:
nit: What does"`1`" of `webgl_materials_texture3d_volume1.html` indicate?

Reply:
I was anticipating for an alternative volume example. But I see that other examples start with a non-numbered file and then add 2,3 etc. Renamed.